models/migrations/v1_18/v230_test.go

```diff
@@ -11,12 +11,16 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func Test_AddConfidentialClientColumnToOAuth2ApplicationTable(t *testing.T) {
-	// premigration
-	type OAuth2Application struct {
-		ID int64
-	}
+// premigration
+type OAuth2Application struct {
```

Comment:
Why another ? There is one in `v1_18/v230.go`

Reply:
They have different fields. This is a part of the tests. At the beginning, there is no newly added column and then add it. So we can not reuse the original struct.

Reply:
c41c6d2515c78e52e79d677e85fefb2893cd2393